### ISSUE TITLE
No feedback mechanism for successful connection

### ISSUE THREAD

**Issue by danrabbit** [elementary/capnet-assist#4]
When connecting to a portal, capnet doesn't automatically close anymore. This means there is no longer a feedback mechanism to let you know that you've successfully logged in to the captive portal.

Launchpad Details: [#LP1610592](https://bugs.launchpad.net/bugs/1610592) Daniel Fore - 2016-08-06 21:01:24 +0000

**Comment by danrabbit** [comment]
Yes,  this happens as a consequence to revision 78, which belongs the  me.  On some captive portals  there is important information displayed on the page after you are logged in,such as credit card transaction information or duration.  It should not close. 

Launchpad Details: [#LPC](https://bugs.launchpad.net/capnet-assist/+bug/1610592/comments/1) Zisu Andrei - 2016-08-06 21:23:16 +0000

**Comment by danrabbit** [comment]
I will get around to removing the alert view from my retires brach,  which brings several improvements,  such as the window not showing when it's not necessary me (which still bugs me as hell) 

Launchpad Details: [#LPC](https://bugs.launchpad.net/capnet-assist/+bug/1610592/comments/2) Zisu Andrei - 2016-08-06 21:24:53 +0000

**Comment by danrabbit** [comment]
Please don't change the status. I'd prefer to have UX team weigh in here 

Launchpad Details: [#LPC](https://bugs.launchpad.net/capnet-assist/+bug/1610592/comments/3) Daniel Fore - 2016-08-06 22:38:41 +0000

**Comment by danrabbit** [comment]
Generally,  the captive portal confirmation page will do that. But I agree with you. 

Launchpad Details: [#LPC](https://bugs.launchpad.net/capnet-assist/+bug/1610592/comments/4) Zisu Andrei - 2016-08-07 09:15:43 +0000

**Comment by cassidyjames** [comment]
It seems like the Android captive portal sign-in automatically closes when there's a connection or when the connection changes (see #5), so I am unsure if networks have "important" information displayed after a connection is made. I'd be curious to see examples of such networks as well as the behavior of other OSes. But I can think of two solutions:

1. Restore the close-on-success flow. This is even explicitly mentioned in the copy at capnet.elementary.io, so it's what I lean towards. It would also resolve the issue where a network appears to be captive and then successfully connects, so you're left with a seemingly random capnet window with that page open.

2. If we want to keep the window open, we could add an infobar with a message like, "You’re connected! You can safely close this window.”

Actually, in combination with the second option, could we automatically close the window _only_ if it successfully loads capnet.elementary.io? That might nicely solve most cases.
